### DISCUSSION TITLE
Render video with DLSS

### DISCUSSION THREAD

**Discussion by cpkcpkcpk** [NVlabs/instant-ngp#824] [Q&A]
Hi, would it be possible to render videos from run.py with DLSS enabled?

Rendering 4K takes at least 20 seconds per frame, so it would be great to have the option to export 1080p upscaled to 4k with DLSS.

**Comment by Tom94** [comment]
Hi there, DLSS is currently somewhat tightly integrated with the render window, so can't easily be used with offline rendering. That said, you could create a 4k window, hide the GUI, set the render resolution to 1080p (and disable dynamic res), enable DLSS, and render your sequence by alternating `testbed.frame()` and `testbed.screenshot()` calls (rather than `testbed.render()`), all through the existing Python bindings.
